app/src/main/java/org/mozilla/fenix/ext/Fragment.kt

```diff
@@ -128,3 +130,14 @@ fun Fragment.removeSecure() {
         WindowManager.LayoutParams.FLAG_SECURE,
     )
 }
+
+/**
+ * Checks if the [Fragment] is on top of the back queue.
+ */
+@SuppressLint("RestrictedApi")
```

Comment:
what's the restricted API here?

Reply:
`displayName`

Reply:
Not sure why they want to restrict it. It is something useful if you don't have any other way to check what fragments are in the back stack.